src/msw/webview_ie.cpp

```diff
@@ -853,25 +857,229 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
-void wxWebViewIE::RunScript(const wxString& javascript)
+wxWebViewIEEmulationLevel wxWebViewIE::GetEmulationLevel()
+{
+    wxRegKey key(wxRegKey::HKCU, _T("SOFTWARE\\Microsoft\\Internet Explorer\\Main\\FeatureControl\\FEATURE_BROWSER_EMULATION"));
+    if ( key.Exists() )
+    {
+        long val = 0;
+
+        if ( !key.QueryValue(wxGetFullModuleName().AfterLast('\\'), &val) )
+        {
+            wxLogWarning(_("Can not get level value of registry key \
```

Comment:
This warning is badly worded and arguably not necessary, what can the end user do about this anyhow and why should he care? If we want to keep it, it should just say "Failed to retrieve the current browser control emulation level".